File: webApi/Models/Estudiante.cs
using System;
using System.Collections.Generic;

namespace webApi.Models;

public partial class Estudiante
{
    public string Carnet { get; set; }

    public int? Clienteid { get; set; }

    public int Cantidaddeviajes { get; set; }

    public int? Millas { get; set; }

}

File: AdminWebApp/src/app/Models/Estudiante.ts
export class Estudiante {
    carnet: string;
    clienteid: number;
    cantidaddeviajes: number;
    millas: number;
  
    constructor(carnet: string, clienteid: number, cantidaddeviajes: number, millas: number) {
      this.carnet = carnet;
      this.clienteid = clienteid;
      this.cantidaddeviajes = cantidaddeviajes;
      this.millas = millas;
    }
  }